README.md

```diff
@@ -35,33 +41,80 @@ To see the graphics, you can open another terminal and run
 ```bash
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
+#### [Grocery Ground Test](python/social_bot/envs/grocery_ground.py)
+```bash
+python3 python/social_bot/envs/grocery_ground.py
+```
 
 ## Environments and Tasks
 
 ### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
 
 * [Goal task](python/social_bot/teacher_tasks.py). 
     
-    To be updated
+    Gif to be updated    
 
 ### [Grocery Ground](python/social_bot/envs/grocery_ground.py)
```

Comment:
We can also show the different robots we have, and briefly introduce them.

Reply:
Yes, we should

Reply:
and also need to briefly describe the logic of teacher-student learning formula.

Reply:
Done